daos/sqlite/Database.php

```diff
@@ -197,4 +211,27 @@ public function optimize() {
             VACUUM;
         ');
     }
+    
+    /**
+     * Initialize 'lastentry' Field in Source table during database upgrade
+     *
+     * @return void
+     */
+    private function initLastEntryFieldDuringUpgrade() { 	
+    	$sources = @\F3::get('db')->exec('SELECT id FROM sources');
+   
+    	// have a look at each entry in the source table
+    	foreach($sources as $current_src) {
+    		//get the date of the newest entry found in the database   		
+    		$latestEntryDate = @\F3::get('db')->exec('SELECT datetime FROM items WHERE source=' . 
+    				                              $current_src['id'] . ' ORDER BY id DESC LIMIT 0, 1');
```

Comment:
Hi, yes I can confirm, ordering by datetime fixes the problem! 

The only scenario I can imagine at the moment when the ordering by id would fail is when the RSS feed was added AFTER the latest post was published and BEFORE running the database upgrade. To stick with your example you must have added the feed today after 13:05 and then implicitly run the database upgrade. 

Is this what happened (or am I missing another scenario?).

If you like I can do a pull request with the fix or you can insert it yourself, as you like.

Thanks to both of you for spotting this!
Martin